theme-switcher/manifest.json

```diff
@@ -0,0 +1,29 @@
+{
+  "browser_action": {
+    "default_title": "Theme switcher",
+    "default_popup": "switcher.html",
+    "browser_style": true,
+    "default_icon": {
+        "128": "star-half.svg"
+    }
+  },
+  "description": "An example of how to use the management API for themes.",
+  "homepage_url": "https://github.com/mdn/webextensions-examples/tree/master/theme-switcher",
+  "manifest_version": 2,
+  "name": "Theme Switcher",
+  "permissions": [
+    "management"
+  ],
+  "icons": {
+      "128": "star-half.svg"
+  },
+  "background": {
```

Comment:
This key should be removed: "background.js" doesn't exist and this will give a warning on install.